src/PeopleWeb.java

```diff
@@ -0,0 +1,159 @@
+        ResultSet rs = stmt.executeQuery();
+        if (rs.next()) {
+            String firstName = rs.getString("first_name");
+            String lastName = rs.getString("last_name");
+            String email = rs.getString("email");
+            String country = rs.getString("country");
+            String ip = rs.getString("ip");
+            person = new Person(id, firstName, lastName, email, country, ip);
+        }
+        return person;
+    }
+
+    public static ArrayList<Person> selectPeople(Connection conn, int offset) throws SQLException {
+        ArrayList<Person> people = new ArrayList<>();
+        PreparedStatement stmt = conn.prepareStatement("SELECT * FROM people LIMIT 20 OFFSET ?");
+        stmt.setInt(1, offset);
+        ResultSet rs = stmt.executeQuery();
+        while (rs.next()) {
+            int id = rs.getInt("id");
+            String firstName = rs.getString("first_name");
+            String lastName = rs.getString("last_name");
+            String email = rs.getString("email");
+            String country = rs.getString("country");
+            String ip = rs.getString("ip");
+            people.add(new Person(id, firstName, lastName, email, country, ip));
+        }
+
+        return people;
+    }
+
+    public static void populateDatabase(Connection conn) throws FileNotFoundException, SQLException {
+        File f = new File("people.csv");
+        Scanner fileScanner = new Scanner(f);
+
+        while (fileScanner.hasNext()) {
+            String line = fileScanner.nextLine();
+            String[] column = line.split(",");
+            if (!column[0].equalsIgnoreCase("id"))
+            insertPerson(conn, column[1], column[2], column[3], column[4], column[5]);
+        }
+    }
+
+    public static void main(String[] args) throws FileNotFoundException, SQLException {
+        Server.createWebServer().start();
+        Connection conn = DriverManager.getConnection("jdbc:h2:./main");
+        createTables(conn);
+        populateDatabase(conn);
+
+        Spark.init();
+
+        Spark.get(
+                "/",
+                ((request, response) -> {
+                    String offset = request.queryParams("offset");
+                    int offsetNum = 0;
+
+                    Integer next1 = null;
+                    Integer next2 = null;
+                    Integer prev1 = null;
+                    Integer prev2 = null;
+
+                    Integer last = null;
+
+                    if (offset != null) {
+                        offsetNum = Integer.parseInt(offset);
+                    }
+
+                    ArrayList<Person> people = selectPeople(conn, offsetNum);
+
+                    Integer prev = null;
+                    if (offsetNum >= 20) {
+                        prev = (offsetNum - 20);
+                        prev1 = prev/20 + 1;
+                    }
+                    Integer next = null;
+                    if (offsetNum < 980) {
```

Comment:
instead of hard-coding 980 everywhere, use a CONSTANT (ie; final static).  
Or, you might want to create a new database method that returns an int and the SELECTS count(*)